### ISSUE TITLE
AutoGraph could not transform

### ISSUE THREAD

**Issue by xuxinyang** [tensorflow/tensorflow#43729]
WARNING: AutoGraph could not transform <function Model.make_test_function.<locals>.test_function at 0x0000022F3CE53430> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert

**Comment by Saduf2019** [comment]
@xuxinyang 
Please refer to [this comment](https://github.com/tensorflow/tensorflow/issues/37144#issuecomment-600350256) and let us know.
Similar issues: #38947 #41120 [link](https://stackoverflow.com/questions/62931610/warningtensorflowautograph-could-not-transform-function-format-example-at)
We see that you have not filled in the issue template, can you please try in the latest tf version(nightly) and let us know if the issue exist.

**Comment by jvishnuvardhan** [comment]
@xuxinyang Can you please share a simple standalone code to reproduce the issue? Thanks!

**Comment by xuxinyang** [comment]
OS: window10 Pro
Development platform:Anaconda
Python: 3.8.3
Tensorflow: 2.2.0
Code:
----------------------------------------------------------------------------------------------------------------------
import tensorflow as tf
import matplotlib.pyplot as plt
import numpy as np

# add dataset fashion——mnist
(train_images, train_labels), (test_images, test_labels) = tf.keras.datasets.fashion_mnist.load_data()
print(train_images.shape)


# image expand dimension
train_images = np.expand_dims(train_images, -1)
test_images = np.expand_dims(test_images, -1)

# build Sequential() model
model = tf.keras.Sequential()
model.add(tf.keras.layers.Conv2D(32, (3,3), input_shape=(28,28,1), activation='relu', padding='same'))
model.add(tf.keras.layers.MaxPooling2D())
model.add(tf.keras.layers.Conv2D(64, (3,3), activation='relu'))
model.add(tf.keras.layers.GlobalAveragePooling2D())
model.add(tf.keras.layers.Dense(10, activation='softmax'))
print(model.output_shape)

model.compile(
    optimizer='adam',
    loss='sparse_categorical_crossentropy',
    metrics=['acc'])

# save model training process
history = model.fit(train_images, train_labels, epochs=5, validation_data=(test_images, test_labels))

**Comment by Saduf2019** [comment]
@xuxinyang 
i ran the code shared on tf 2.2 and tf nightly and do not face any errors please find the [gist here for nightly](https://colab.research.google.com/gist/Saduf2019/7409ab49eca8b5fe7cbac89e4153349b/untitled418.ipynb) and [tf 2.2](https://colab.research.google.com/gist/Saduf2019/d8c5bcf1a7b5fed844f6fce3fa9a628f/untitled424.ipynb).
Could you let us know if you have gone through the link shared, and try your code on nightly and let us know if the issue exist.

**Comment by xuxinyang** [comment]
OK! I ran the code on nightly, the issue not exist! Thank you! Maybe some other factors！

**Comment by Saduf2019** [comment]
@xuxinyang 
Glad the issue is resolved for you, can you please move this to closed status, thanks for your update!
